pybutton/resources/resource.py

```diff
@@ -31,10 +31,9 @@ class Resource(object):
 
     '''
 
-    API_BASE = 'https://api.usebutton.com'
-
-    def __init__(self, api_key):
+    def __init__(self, api_key, config):
         self.api_key = api_key
+        self.config = config
```

Comment:
even though you're going to end up repeating it a lot, I think a docstring here explaining `config` makes sense (especially given all the keys are required at this point)